verilog/CST/module.h

```diff
@@ -30,13 +30,25 @@
 
 namespace verilog {
 
-template <typename T0, typename T1, typename T2, typename T3, typename T4,
-          typename T5, typename T6, typename T7>
-verible::SymbolPtr MakeModuleHeader(T0&& keyword, T1&& lifetime, T2&& id,
-                                    T3&& imports, T4&& parameters, T5&& ports,
-                                    T6&& attribute, T7&& semi) {
+template <typename T0,
```

Comment:
This formatting looks weird. Doesn't seem like google code style. Can you revert this? (maybe fix your formatter?)

Also, formatting changes of unrelated code (which you didn't directly touch) are usually not welcome (clutters the code review).